src/_includes/assets/css/screen.css

```diff
@@ -2389,10 +2400,13 @@ a.banner:hover span {
   }
 
   .ad-layout .sidebar {
+    background-color: var(--gray05);
```

Comment:
This makes the entire side bar gray.

Is that the intended behavior? Seems like we have a few different ways of loading GAM ads, adding in the advertisement text, and setting the background color.

Not a huge deal at all, but we might want to try consolidating the different methods in the future, or consider removing the separate background color like on the forum.

Reply:
Yes, I am testing different methods to see which ones does better.